tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -847,7 +851,7 @@
         border-left: 1px solid lightgrey;
       }
     </style>
-    <paper-dialog id="inferencesettings" class="inference-settings" opened="true" with-backdrop>
+    <paper-dialog id="inferencesettings" class$="[[getInferenceSettingsClass_(local)]]" opened="true">
```

Comment:
should `opened` be based on `local`?

Reply:
good idea. changed, and changed class back to inference-settings